.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,16 @@
+on:
```

Comment:
Here you could say `on: push` to trigger workflow on any push to any branch. Thus perhaps the qualifier `*` for branches is not needed?